src/sentry/static/sentry/app/stores/organizationStore.jsx

```diff
@@ -0,0 +1,30 @@
+import Reflux from 'reflux';
+
+import OrganizationActions from 'app/actions/organizationActions';
+import OrganizationsActions from 'app/actions/organizationsActions';
+
+const OrganizationStore = Reflux.createStore({
+  listenables: OrganizationActions,
```

Comment:
I would instead call `this.listenTo` like you do with `OrganizationsAction.update` because it's a bit more explicit. E.g. as a reader without too much Reflux knowledge you're able to read `this.listenTo()` and mostly know what it's intended to do, `listenables` is a bit too much magic and requires more Reflux knowledge.